### PR TITLE
Replace nullptr references with NULL

### PR DESCRIPTION
**Pull request by FranklandJack**
Unified Runtime is a C API. There is no `nullptr` in C but the spec and
headers reference the `nullptr` value. Replace these reference with
`NULL` which is the correct null pointer value in C.

Fixes https://github.com/oneapi-src/unified-runtime/issues/11.

### CODE REVIEW

**Comment by FranklandJack** [comment]
Fixes https://github.com/oneapi-src/unified-runtime/issues/11.

File: include/zer_api.h [hunk 1]
```diff
@@ -6820,4 +6820,4 @@ typedef struct _zer_callbacks_t
 } // extern "C"
 #endif
 
-#endif // _ZER_API_H
+#endif // _ZER_API_H
```
Comment (kbenzie) [inline include/zer_api.h:6823]: Is this removing the trailing new line?

Comment (FranklandJack) [reply include/zer_api.h:6823]: It is yes.

Comment (kbenzie) [reply include/zer_api.h:6823]: Would be nice if the generator script was consistent...

**Review by AerialMantis — APPROVED** [review]
LGTM.

**Review by kbenzie — APPROVED** [review]

**Pull request merged**